### ISSUE TITLE
DDLDatabase failing to load and parse SQL files

### ISSUE THREAD

**Issue by dustpuppyNGTI** [jOOQ/jOOQ#7766]
I'm trying to setup the DDLDatabase to generate my code. I have two problems here. First the wildcard file loading fails. Second the files themselves can not be parsed.

When I configure the generator with this:

    <value>src/main/resources/db/migration/*.sql</value>

No files are actually parsed and no code is generated. When I configure it this way:

    <value>src/main/resources/db/migration/V1.1__initial.sql</value>

The generator fails with this error:

[ERROR] Failed to execute goal org.jooq:jooq-codegen-maven:3.11.4:generate (default) on project generator-test: Error running jOOQ code generation tool: Unexpected token or missing query delimiter: [3:207] ...FAULT NULL, PRIMARY KEY (`box_id`)) ENGINE=InnoDB [*]DEFAULT CHARSET=utf8mb4;
[ERROR] CREATE TABLE `endpoint` ( `endpoint_id` int(10) unsigne...
[ERROR] -> [Help 1]

The SQL itself is accepted as valid by MariaDB.

I've setup an example project here: https://github.com/dustpuppyNGTI/jooqgeneratortest
Just run 'mvn package' from the root.

**Comment by lukaseder** [comment]
Thanks a lot for your help in reproducing this issue. I can already confirm the part about the path containing the asterisk. The problem is that currently, this path should be fully qualified. E.g. when using maven, prepend `${basedir}` to your path:

```xml
<value>${basedir}/src/main/resources/db/migration/*.sql</value>
```

There are other problems with those wildcards, which I will fix as part of: #7767.

Will get back to the parser issue soon.

**Comment by lukaseder** [comment]
Wildcard problem fixed for jOOQ 3.12: #7767. Backport scheduled for 3.11.5: #7768

**Comment by lukaseder** [comment]
I've created a separate parser bugs:

- Unsupported `DEFAULT CHARSET` clause: #7769
- Unsupported named and inlined unique index specifications: #7268

With this, each individual problem can be tracked separately, and I will close this particular issue as a duplicate of the other ones.

Thanks again for reporting!
